crates/ide-diagnostics/src/tests.rs

```diff
@@ -68,6 +68,91 @@ fn check_nth_fix(nth: usize, ra_fixture_before: &str, ra_fixture_after: &str) {
     assert_eq_text!(&after, &actual);
 }
 
+pub(crate) fn check_fixes_unordered(ra_fixture_before: &str, ra_fixtures_after: Vec<&str>) {
+    for ra_fixture_after in ra_fixtures_after.iter() {
+        check_has_fix(ra_fixture_before, ra_fixture_after)
+    }
+}
+
+#[track_caller]
+pub(crate) fn check_has_fix(ra_fixture_before: &str, ra_fixture_after: &str) {
```

Comment:
whats different between this and `check_fix`?

Reply:
`check_fix` expects the desired fix to be the first suggested fix (or there to be only 1 fix in most cases).
This doesn't care about the ordering, the fix just has to be there somewhere.

I added this as sepparate method as I wasn't sure if `check_fix` should or shouldn't care where in the list of fixes the desired fix is